[2024-06-06T21:33:28.011+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T21:33:28.249+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_701276 scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T21:33:28.275+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_701276 scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T21:33:28.276+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-06-06T21:33:28.342+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_index_id_701276> on 2024-06-05 00:00:00+00:00
[2024-06-06T21:33:28.353+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'load_data_mb', 'load_index_id_701276', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/load_data_mb.py', '--cfg-path', '/tmp/tmpcc173mli']
[2024-06-06T21:33:28.358+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask load_index_id_701276
[2024-06-06T21:33:28.360+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=718) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-06T21:33:28.362+0000] {standard_task_runner.py:63} INFO - Started process 719 to run task
[2024-06-06T21:33:28.467+0000] {task_command.py:426} INFO - Running <TaskInstance: load_data_mb.load_index_id_701276 scheduled__2024-06-05T00:00:00+00:00 [running]> on host e6f094d2943c
[2024-06-06T21:33:28.714+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_data_mb' AIRFLOW_CTX_TASK_ID='load_index_id_701276' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-05T00:00:00+00:00'
[2024-06-06T21:33:28.716+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T21:33:50.656+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T21:33:50.657+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 2 column 1 (char 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/load_data_mb.py", line 26, in load_index
    for status in response.json()["info"]:
                  ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 2 column 1 (char 1)
[2024-06-06T21:33:50.683+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=load_data_mb, task_id=load_index_id_701276, execution_date=20240605T000000, start_date=20240606T213328, end_date=20240606T213350
[2024-06-06T21:33:50.706+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 5 for task load_index_id_701276 (Expecting value: line 2 column 1 (char 1); 719)
[2024-06-06T21:33:50.734+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-06T21:33:50.759+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T21:33:50.767+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-06T22:52:02.562+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T22:52:02.604+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_701276 scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T22:52:02.615+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_701276 scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T22:52:02.616+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-06-06T22:52:02.635+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_index_id_701276> on 2024-06-05 00:00:00+00:00
[2024-06-06T22:52:02.644+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'load_data_mb', 'load_index_id_701276', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/load_data_mb.py', '--cfg-path', '/tmp/tmpuv04a89z']
[2024-06-06T22:52:02.647+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2518) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-06T22:52:02.648+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask load_index_id_701276
[2024-06-06T22:52:02.649+0000] {standard_task_runner.py:63} INFO - Started process 2519 to run task
[2024-06-06T22:52:02.712+0000] {task_command.py:426} INFO - Running <TaskInstance: load_data_mb.load_index_id_701276 scheduled__2024-06-05T00:00:00+00:00 [running]> on host e6f094d2943c
[2024-06-06T22:52:02.824+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_data_mb' AIRFLOW_CTX_TASK_ID='load_index_id_701276' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-05T00:00:00+00:00'
[2024-06-06T22:52:02.825+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T22:52:15.542+0000] {logging_mixin.py:188} INFO - {'status': 'success', 'info': [{'index_name': 'валовой сбор отдельных сельскохозяйственных культур в весе после доработки', 'index_id': 701276, 'dic_names': ['Регионы', 'Виды земель', 'Категории сельхозтоваропроизводителей', 'Растениеводство'], 'period_name': 'Год', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'валовой сбор отдельных сельскохозяйственных культур в весе после доработки', 'index_id': 701276, 'dic_names': ['Регионы', 'Формы и видов собственности', 'Виды земель', 'Категории сельхозтоваропроизводителей', 'Растениеводство'], 'period_name': 'Один раз в год', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'валовой сбор отдельных сельскохозяйственных культур в весе после доработки', 'index_id': 701276, 'dic_names': ['Регионы', 'Размерность юридических лиц, филиалов и представительств, а также субъектов индивидуального предпринимательства по численности работников', 'Виды земель', 'Категории сельхозтоваропроизводителей', 'Растениеводство'], 'period_name': 'Один раз в год', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'валовой сбор отдельных сельскохозяйственных культур в весе после доработки', 'index_id': 701276, 'dic_names': ['Регионы', 'Виды земель', 'Категории сельхозтоваропроизводителей', 'Растениеводство'], 'period_name': 'Один раз в год', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'валовой сбор отдельных сельскохозяйственных культур в весе после доработки', 'index_id': 701276, 'dic_names': ['Регионы', 'Виды земель', 'Категории сельхозтоваропроизводителей', 'Сектора экономики', 'Растениеводство'], 'period_name': 'Один раз в год', 'info': 'ошибка талдау', 'error_code': 500}, {'index_name': 'валовой сбор отдельных сельскохозяйственных культур в весе после доработки', 'index_id': 701276, 'dic_names': ['Регионы', 'Организационно-правовые формы хозяйствования', 'Виды земель', 'Категории сельхозтоваропроизводителей', 'Растениеводство'], 'period_name': 'Один раз в год', 'info': 'ошибка талдау', 'error_code': 500}]}
[2024-06-06T22:52:15.544+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-06T22:52:15.545+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T22:52:15.557+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=load_data_mb, task_id=load_index_id_701276, execution_date=20240605T000000, start_date=20240606T225202, end_date=20240606T225215
[2024-06-06T22:52:15.597+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T22:52:15.616+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T22:52:15.622+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
